client/my-sites/themes/style.scss

```diff
@@ -253,3 +273,30 @@
 		background: linear-gradient( to bottom, transparentize( white, 0.5 ), transparent );
 	}
 }
+
+@include breakpoint( ">1040px" ) {
```

Comment:
It's ok. One day we'll be able to do that contextual on the actual width of the component instead of the global viewport width, but for today... I think it's effective. Not ideal, but hey, works. Maybe let's add a one-line comment in the code to explain why, given the question might come up:

> Tweaked breakpoint because this component is used in a two-columns layout and gets narrower earlier.